app/frontend/src/common/assets/js/leaflet-areaselect.js

```diff
@@ -0,0 +1,184 @@
+L.AreaSelect = L.Class.extend({
```

Comment:
need a blurb about the attribution of this file and the License (I think this is MIT licensed if I'm looking at the right repo)

Reply:
where do you want the blurb? also what should it say?

Reply:
```suggestion
/** 
 * leaflet-areaselect plugin by Jonatan Heyman
 * https://github.com/heyman/leaflet-areaselect
 * Source code made available under MIT license.
 */

L.AreaSelect = L.Class.extend({
```